docs/README-for-packages

```diff
@@ -18,3 +18,7 @@ An example configuration file is provided in the same directory as
 this README. Copy it to /etc/rabbitmq/rabbitmq.config to use it. The
 RabbitMQ server must be restarted after changing the configuration
 file.
+
+An example policy file for HA queues is provided in the same directory
```

Comment:
If you change that PR one more time, I suggest to put the exact example name here: set_rabbitmq_policy.sh.example, so that people know what to search

BTW since you removed .sh suffix from the script, it might be removed from the example as well